org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/real/ExampleMalcolmDeviceTest.java

```diff
@@ -358,15 +362,15 @@ public void configureAndRunDummyMalcolm() throws Exception {
 			assertEquals(seekStructure, seekCall.getStructure());
 			assertEquals(seekPVStructure, seekCall);
 
-			modelledDevice.dispose();
+			malcolmDevice.dispose();
 
 		} catch (Exception ex) {
 			ex.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)